src/main/java/ui/UI.java

```diff
@@ -443,6 +448,27 @@ private void primaryRepoChanged(String repoId) {
         logic.openPrimaryRepository(repoId);
         logic.setDefaultRepo(repoId);
     }
+    
+    public void switchDefaultRepo(){
+        logger.info("Triggered switch default repo");
+        String[] openRepos = logic.getOpenRepositories().toArray(new String[0]);
```

Comment:
But the combo box only allows you to get one open repository using `getText`, right?